cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/ReadThroughBookCache.java

```diff
@@ -0,0 +1,130 @@
+package com.exactpro.cradle.cassandra;
```

Comment:
Add copyright.

Reply:
Added missing copyrights in [ef1177a](https://github.com/th2-net/cradleapi/pull/85/commits/ef1177aad13d8ec407195a41af6e5178a5943b90)